editor/editor_quick_open.cpp

```diff
@@ -123,6 +123,7 @@ void EditorQuickOpen::_update_search() {
 		for (int i = 0; i < entry_limit; i++) {
 			TreeItem *ti = search_options->create_item(root);
 			ti->set_text(0, entries[i].path);
+			ti->set_icon_max_width(0, 16 * EDSCALE);
```

Comment:
You should use `class_icon_size` theme constant.
But don't use `get_theme_constant()` inside the loop.

Reply:
Thanks for the hint! But I'm not sure if have to multiply it by `EDSCALE`

Reply:
It shouldn't be multiplied.